java/Oleksii-Riabov/src/com/zoolatech/lecture2/tasks/_1/Calculator.java

```diff
@@ -0,0 +1,134 @@
+    }
+
+    public static void numbersEvaluationMethod(double number1, double number2, char c) {
+        switch (c) {
+            case '+' -> addition(number1, number2);
+            case '-' -> subtraction(number1, number2);
+            case '*' -> multiplication(number1, number2);
+            case '/' -> division(number1, number2);
+            default -> throw new IllegalArgumentException("Enter symbol of an operation");
+        }
+    }
+
+    public static void fractionalPartCheck(double number) {
+        if (number % 1 == 0) {
+            System.out.println((int) number);
+        } else {
+            System.out.println(number);
+        }
+    }
+
+    public static void addition (double number1, double number2) {
+        fractionalPartCheck(number1 + number2);
+    }
+
+    public static void subtraction (double number1, double number2) {
+        fractionalPartCheck(number1 - number2);
+    }
+
+    public static void multiplication (double number1, double number2) {
+        fractionalPartCheck(number1 * number2);
+    }
+
+    public static void division (double number1, double number2) {
+        if (number2 == 0) {
+            throw new ArithmeticException("Divide by zero is not allowed!");
+        } else {
+            fractionalPartCheck(number1 / number2);
+        }
+    }
+
+    public static double doubleValue() {
+        while (true) {
+            Scanner scanner = new Scanner(System.in);
+            if (scanner.hasNextDouble()) {
+                return scanner.nextDouble();
+            } else {
+                System.out.println("Enter numeric value");
+                continue;
+            }
+        }
+    }
+
+    public static void instructions() {
+        System.out.println("""
+                To add values enter "+"
+                To subtract values enter "-"
+                To multiply values enter "*"
+                To divide values enter "/"
+                To show first value enter "1"
+                To show second value enter "2"
+                To show instructions enter "i"
+                To end program enter "exit"
+                """);
+    }
+
+    public static void main(String[] args) {
+        System.out.println("Define first value");
+        Calculator calculator = new Calculator(doubleValue());
+        System.out.println("First value is " + calculator.getValue()+ "\n");
+
+        System.out.println("Define second value");
+        double secondValue = doubleValue();
+        System.out.println("Second value is " + secondValue+ "\n");
+
+        Scanner input = new Scanner(System.in);
+
+        instructions();
+        outer:
+        while (true) {
+//            instructions();
+            System.out.println("Enter evaluation sign to evaluate values");
+
+            String line = input.nextLine().toLowerCase();
+            switch (line) {
+                case "+" -> {
+                    numbersEvaluationMethod(calculator.getValue(), secondValue, '+');
```

Comment:
The calculator should be called like this:
```
calculator. addition(value)
```

Reply:
Already update this method